pkg/controller/complianceremediation/complianceremediation_controller_test.go

```diff
@@ -41,20 +42,24 @@ func getMockedRemediation(name string, labels map[string]string, applied bool, s
 		},
 	}
 
+	ign := igntypes.Config{
+		Ignition: igntypes.Ignition{
+			Version: igntypes.MaxVersion.String(),
```

Comment:
Is this necessary since in the non-test code we don't set the version either? `make test-unit` seems to pass with the version removed.

Reply:
Let me remove it.

Reply:
Done